db/migrate/20140409134713_move_log_collection_depot_settings_to_file_depot.rb

```diff
@@ -5,20 +5,14 @@ class Authentication < ActiveRecord::Base
 
   class Configuration < ActiveRecord::Base
     serialize :settings, Hash
-    self.inheritance_column = :_type_disabled # disable STI
   end
 
-  class FileDepot < ActiveRecord::Base
-    self.inheritance_column = :_type_disabled # disable STI
```

Comment:
but when was this migration created the column was not here, so should I respect state of DB in that time or state of latest DB ?